Objects/Objects/BuiltElements/Duct.cs

```diff
@@ -177,7 +185,7 @@ public class RevitFlexDuct : RevitDuct
     /// <param name="parameters"></param>
     /// <remarks>Assign units when using this constructor due to <paramref name="width"/>, <paramref name="height"/>, and <paramref name="diameter"/> params</remarks>
     [SchemaInfo("RevitFlexDuct", "Creates a Revit flex duct", "Revit", "MEP")]
-    public RevitFlexDuct(string family, string type, [SchemaMainParam] ICurve baseCurve, string systemName, string systemType, Level level, double width, double height, double diameter, Vector startTangent, Vector endTangent, double velocity = 0, List<Parameter> parameters = null)
+    public RevitFlexDuct(string family, string type, [SchemaMainParam] ICurve baseCurve, string systemName, string systemType, Level level, double width, double height, double diameter, Vector startTangent, Vector endTangent, double velocity = 0, List<Parameter> parameters = null, Connector[] connectors = null)
```

Comment:
A new constructor needs to be added with the `connectors` argument to not break backwards compatibility with existing GH components